agent/rpc/middleware/rate_limit_mappings.go

```diff
@@ -0,0 +1,144 @@
+	"FederationState.ListMeshGateways": rate.OperationTypeRead,
+
+	"Health.ChecksInState": rate.OperationTypeRead,
+	"Health.NodeChecks":    rate.OperationTypeRead,
+	"Health.ServiceChecks": rate.OperationTypeRead,
+	"Health.ServiceNodes":  rate.OperationTypeRead,
+
+	"Intention.Apply": rate.OperationTypeWrite,
+	"Intention.Check": rate.OperationTypeRead,
+	"Intention.Get":   rate.OperationTypeRead,
+	"Intention.List":  rate.OperationTypeRead,
+	"Intention.Match": rate.OperationTypeRead,
+
+	"Internal.CatalogOverview":               rate.OperationTypeRead,
+	"Internal.EventFire":                     rate.OperationTypeWrite,
+	"Internal.ExportedPeeredServices":        rate.OperationTypeRead,
+	"Internal.ExportedServicesForPeer":       rate.OperationTypeRead,
+	"Internal.GatewayIntentions":             rate.OperationTypeRead,
+	"Internal.GatewayServiceDump":            rate.OperationTypeRead,
+	"Internal.IntentionUpstreams":            rate.OperationTypeRead,
+	"Internal.IntentionUpstreamsDestination": rate.OperationTypeRead,
+	"Internal.KeyringOperation":              rate.OperationTypeRead,
+	"Internal.NodeDump":                      rate.OperationTypeRead,
+	"Internal.NodeInfo":                      rate.OperationTypeRead,
+	"Internal.PeeredUpstreams":               rate.OperationTypeRead,
+	"Internal.ServiceDump":                   rate.OperationTypeRead,
+	"Internal.ServiceGateways":               rate.OperationTypeRead,
+	"Internal.ServiceTopology":               rate.OperationTypeRead,
+
+	"KVS.Apply":    rate.OperationTypeWrite,
+	"KVS.Get":      rate.OperationTypeRead,
+	"KVS.List":     rate.OperationTypeRead,
+	"KVS.ListKeys": rate.OperationTypeRead,
+
+	"Operator.AutopilotGetConfiguration": rate.OperationTypeRead,
+	"Operator.AutopilotSetConfiguration": rate.OperationTypeWrite,
+	"Operator.AutopilotState":            rate.OperationTypeRead,
+	"Operator.RaftGetConfiguration":      rate.OperationTypeRead,
+	"Operator.RaftRemovePeerByAddress":   rate.OperationTypeWrite,
+	"Operator.RaftRemovePeerByID":        rate.OperationTypeWrite,
+	"Operator.ServerHealth":              rate.OperationTypeRead,
+
+	"PreparedQuery.Apply":         rate.OperationTypeWrite,
+	"PreparedQuery.Execute":       rate.OperationTypeRead,
+	"PreparedQuery.ExecuteRemote": rate.OperationTypeRead,
+	"PreparedQuery.Explain":       rate.OperationTypeRead,
+	"PreparedQuery.Get":           rate.OperationTypeRead,
+	"PreparedQuery.List":          rate.OperationTypeRead,
+
+	"Session.Apply":        rate.OperationTypeWrite,
+	"Session.Check":        rate.OperationTypeRead,
+	"Session.Get":          rate.OperationTypeRead,
+	"Session.List":         rate.OperationTypeRead,
+	"Session.NodeSessions": rate.OperationTypeRead,
+	"Session.Renew":        rate.OperationTypeWrite,
+
+	"Status.Leader":    rate.OperationTypeRead,
+	"Status.Peers":     rate.OperationTypeRead,
+	"Status.Ping":      rate.OperationTypeRead,
+	"Status.RaftStats": rate.OperationTypeRead,
```

Comment:
Same here we need to bypass those, those are used by autopilot, which is considered as vital to the cluster.

Reply:
Updated